bool question
---
bool answer
nav_msgs/OccupancyGrid newGrid
